client/pages/Interests/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import InterestCard from '../../components/cards/InterestCard';
+/**
+ * @description allows users to select their interests
+ *
+ * @class Interests
+ * @extends {React.Component}
+ */
+class Interests extends React.Component {
+  state = {
+    interests: [
```

Comment:
Nice move with the fixtures. I think it will be better if we move it into a separate file as it is just dummy data and not officially a part of the definition of this component.

Reply:
Thank you for your feedback. This has been fixed.